docs/orchestrating-elastic-stack-applications/beat.asciidoc

```diff
@@ -362,6 +366,90 @@ rules:
 
 To deploy Beats in clusters with the Pod Security Policy admission controller enabled, or in OpenShift clusters, you must grant additional permissions to the Service Account used by the Beat Pods. Those Service Accounts must be bound to a Role or ClusterRole that has `use` permission for the required Pod Security Policy or Security Context Constraints. Different Beats and their features might require different settings set in their PSP/SCC.
 
+
+[id="{p}-beat-configuration-examples"]
+== Configuration Examples
+
+Below you can find manifests that address a number of common use cases and can be your starting point in exploring Beats deployed with ECK. These manifests are self-contained and work out-of-the-box on any non-secured Kubernetes cluster. All of them contain three-node Elasticsearch cluster and single Kibana instance. All Beat configurations set up Kibana dashboards if they are available for a given Beat and all required RBAC resources.
+
+[source,sh,subs="attributes"]
+=== Metricbeat for Kubernetes monitoring
+
+[source,sh,subs="attributes"]
+----
+kubectl apply -f {beats_url}/metricbeat_hosts.yaml
+----
+
+
+Deploys Metricbeat as a DaemonSet that monitors the host resource usage (cpu, memory, network, filesystem) and Kubernetes resources (Nodes, Pods, Containers, Volumes).
+
+=== Filebeat with autodiscover
+
+[source,sh,subs="attributes"]
+----
+kubectl apply -f {beats_url}/filebeat_autodiscover.yaml
+----
+
+Deploys Filebeat as DaemonSet with autodiscover feature enabled. All pods in all namespaces will have logs shipped to an Elasticsearch cluster.
```

Comment:
```suggestion
Deploys Filebeat as a DaemonSet with the autodiscover feature enabled. It will collect logs from pods in every namespace and load them to the connected Elasticsearch cluster.
```